.thoth.yaml

```diff
@@ -1,14 +1,80 @@
 host: khemenu.thoth-station.ninja
 tls_verify: false
 requirements_format: pipenv
+overlays_dir: overlays
 
 runtime_environments:
-  - name: rhel:8
+  - name: solver-fedora-30-py36
     operating_system:
-      name: rhel
+      name: "fedora"
+      version: "30"
+    python_version: "3.6"
+    recommendation_type: latest
+    base_image: quay.io/thoth-station/s2i-thoth-f30-py36:v0.23.2
```

Comment:
Some of the s2i base images are not available, we should provide them to have the whole workflow in place.

Reply:
++
yes, we should create them